nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -8,6 +8,30 @@ with lib;
 {
   imports = [ ./installation-cd-base.nix ];
 
+  users.extraUsers.live = {
+    isNormalUser = true;
+    uid = 1000;
+    extraGroups = [ "wheel" "networkmanager" "video" ];
```

Comment:
Are `networkmanager` and `video` needed? NetworkManager should work for locally logged-in users without it, and likewise logged-in users get access for /dev/video* via an ACL.

Reply:
I'm not sure. I think I'd have an issue on my own installation if I didn't use this. I've received guidance that this should be done, but I've yet to verify this.

I will find out :+1:

Reply:
I tried this in a vm and it caused it to require authentication to initiate a connection with networkmanager. Not sure about `video` though but perhaps it's fine to leave it.